tensorboard/webapp/app_routing/programmatical_navigation_types.ts

```diff
@@ -40,10 +40,17 @@ export interface NavigateToExperiments {
   resetNamespacedState?: boolean;
 }
 
+export interface NavigateToFlags {
+  routeKind: RouteKind.FLAGS;
+  routeParams: {},
+  resetNamespacedState?: boolean;
+}
+
 export type ProgrammaticalNavigation =
   | NavigateToExperiment
   | NavigateToCompare
-  | NavigateToExperiments;
+  | NavigateToExperiments
+  | NavigateToFlags;
```

Comment:
This is not technically necessary but it does make the typing more accurate.

Reply:
I don't have strong feelings about this either way. However, I would have left this out because we do not want to programmatically navigate. 

I can definitely see this being useful one day if we decide to have a secret button that navigates here or something. But, I think we can add it then.